web/src/engine/osk/src/input/gestures/browser/subkeyPopup.ts

```diff
@@ -59,6 +59,16 @@ export default class SubkeyPopup implements GestureHandler {
       this.clear();
     });
 
+    source.on('stage', () => {
```

Comment:
What does this do? Can you add a comment?

Reply:
```suggestion
    // When subkey-selection is fully triggered, emit the selected key.
    source.on('stage', () => {
```